modules/insights/private-link-scope/main.bicep

```diff
@@ -6,6 +6,12 @@ metadata owner = 'Azure/module-maintainers'
 @minLength(1)
 param name string
 
+@description('''Optional. Specifies the access mode of ingestion or queries through associated private endpoints in scope. For security reasons, it is recommended to use PrivateOnly whenever possible to avoid data exfiltration.
+
+  * Private Only (default) - Allows the connected virtual network to reach only Private Link resources. This mode is the most secure.
```

Comment:
Just one last comment from my end @johnlokerse.. the default is private only if private endpoints are configured. Can you please update the description to reflect the behaviour. For example if we deploy the "default" test case, the behaviour will be 'open'. Other than that I am good.. thank you so much for your contribution <3

Reply:
Also rerunning static tests just to be sure...